includes/wikia/tests/core/WikiaBaseTest.class.php

```diff
@@ -92,7 +92,7 @@ protected function setUp() {
 		if ( $this->mockProxy !== null ) {
 			throw new Exception("Previous test did not execute tearDown()");
 		}
-		$this->mockProxy = new WikiaMockProxy();
+		$this->mockProxy = function_exists('runkit_method_rename') ? new WikiaMockProxy() : new WikiaMockProxyUopz();
```

Comment:
Imagine having php5.6 without runkit but with uopz enabled. I'd rather base this check on php version